### PR TITLE
Refactor save/cancel buttons into activity component so resuable and add save event

### PR DESCRIPTION
**Pull request by mdgbayly**
First step for implementing save/cancel - add a save event.

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-editor-footer.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import '../d2l-activity-editor-buttons.js';
```
Comment (wongvincent) [inline components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-editor-footer.js]: No longer needs to import the localization, LocalizeMixin, and RtlMixin

Comment (mdgbayly) [reply components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-editor-footer.js]: will clean up on next PR

File: components/d2l-activity-editor/d2l-activity-assignment-editor/lang/en.js [hunk 2]
```diff
@@ -13,8 +13,6 @@ export default {
 	"hlpTurnitin": "The Turnitin® Integration allows you to use Originality Check® to assess student work for academic integrity and to use the GradeMark® external evaluation tool for assessment.", // turnitin help
 	"btnEditTurnitin": "Manage Turnitin® Integration", // edit turnitin button
 	"btnCloseDialog": "Close this Dialog", // close dialog button
-	"btnCancel": "Cancel", // cancel button
```
Comment (ppaskaris-d2l) [inline components/d2l-activity-editor/d2l-activity-assignment-editor/lang/en.js]: This broke the Turnitin dialog

Comment (mdgbayly) [reply components/d2l-activity-editor/d2l-activity-assignment-editor/lang/en.js]: @ppaskaris-d2l - oops, sorry, didn't even think to look if something else was using it, but should have done.

**Pull request merged**